packages/@react-spectrum/list/src/styles.css

```diff
@@ -489,6 +489,15 @@
   border-color: var(--spectrum-global-color-blue-500);
   background-color: var(--spectrum-alias-highlight-selected);
   box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
+
+  &.react-spectrum-ListView--emphasized {
+    .react-spectrum-ListViewItem::after {
+        inset-inline-start: 1px;
+        inset-inline-end: 1px;
+        inset-block-end: 1px;
+        inset-block-start: 1px;
+    }
+  }
```

Comment:
so this will change the size of all selection borders, which will result in thicker lines where they intersect with the listview's overall border, it will also cause a small wiggle because that selection border will jump inwards 1px when you hover

I think a better solution would be to alter the dropTarget ring. Something more along these lines. I did not test this at all. Reason for the two is one is the actual border of the ListView, the other is the inner one that covers all the border intersections since it comes last in dom order
```
.react-spectrum-ListView--dropTarget {
  border-color: var(--spectrum-global-color-blue-500);
  background-color: var(--spectrum-alias-highlight-selected);
  box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
  &::after {
    border-radius: inherit;
    border-color: var(--spectrum-global-color-blue-500);
    background-color: var(--spectrum-alias-highlight-selected);
    box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
    content: '';
    display: block;
    position: absolute;
    top: 0;
    inset-inline-start: 0;
    width: 100%;
    height: 100%;
  }
}
```

Reply:
Great idea, that works way better. Just pushed that up